libs/product/driver/magento/src/product-driver.module.ts

```diff
@@ -21,7 +22,7 @@ import { DAFF_MAGENTO_GET_A_PRODUCT_QUERY_NAME } from './queries/get-product';
   ],
 })
 export class DaffProductMagentoDriverModule {
-  static forRoot(config: DaffProductMagentoDriverConfig): ModuleWithProviders<DaffProductMagentoDriverModule> {
+  static forRoot(config: DaffProductMagentoDriverConfig = MAGENTO_PRODUCT_CONFIG_DEFAULT): ModuleWithProviders<DaffProductMagentoDriverModule> {
```

Comment:
Shouldn't the default be provided in the token definition?

Reply:
I added it to the token def but I did not remove it as a default arg here.